Test/Integration/Model/Transaction/OrderTest.php

```diff
@@ -15,172 +15,128 @@
  * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  */
 
-// @codingStandardsIgnoreStart
-
 namespace Taxjar\SalesTax\Test\Integration\Model\Transaction;
 
-use Magento\InventoryReservationsApi\Model\CleanupReservationsInterface;
-use Magento\Sales\Model\Order;
-use Magento\TestFramework\Helper\Bootstrap;
+use Taxjar\SalesTax\Model\Transaction\Order as TaxjarOrder;
+use Taxjar\SalesTax\Test\Integration\IntegrationTestCase;
+use Taxjar\SalesTax\Util\Fixtures\Catalog\ProductBuilder;
+use Taxjar\SalesTax\Util\Fixtures\Customer\AddressBuilder;
+use Taxjar\SalesTax\Util\Fixtures\Customer\CustomerBuilder;
+use Taxjar\SalesTax\Util\Fixtures\Sales\OrderBuilder;
 
 /**
- * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
- * @see https://app.hiptest.com/projects/69435/test-plan/folders/419534/scenarios/2587535
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)/**
+ *
+ * @magentoDbIsolation enabled
+ * @magentoAppIsolation enabled
  */
-class OrderTest extends \PHPUnit\Framework\TestCase
+class OrderTest extends IntegrationTestCase
 {
     /**
-     * @var CleanupReservationsInterface
-     */
-    protected $cleanupReservations;
-
-    /**
-     * @var Order
+     * @var TaxjarOrder
      */
-    protected $order;
-
-    /**
-     * @var \Taxjar\SalesTax\Model\Transaction\Order
-     */
-    protected $transactionOrder;
+    protected $taxjarOrder;
 
     protected function setUp(): void
     {
-        $this->cleanupReservations = Bootstrap::getObjectManager()->get(CleanupReservationsInterface::class);
-        $this->order = Bootstrap::getObjectManager()->get(Order::class);
-        $this->transactionOrder = Bootstrap::getObjectManager()->get('Taxjar\SalesTax\Model\Transaction\Order');
+        $this->taxjarOrder = $this->objectManager->get(TaxjarOrder::class);
     }
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple.php
-     */
     public function testDefaultOrder()
```

Comment:
Perhaps we should rename some of these tests to better indicate what behavior we are testing. Maybe something like: testTotalsAreCorrect ? Though that doesn't quite fit since we are also testing the provider parameter here. Something to think about anyway.